manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS2.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GitTools.GitVersion
+PackageVersion: 5.11.1
+InstallerType: zip
+Commands:
+- gitversion
+ReleaseDate: 2022-11-08
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Git.Git
Installers:
```

Reply:
Hi @arturcic, 

This package appears to depend on Git to operate. When launched, it gives this error:
> Cannot find the .git directory